### DISCUSSION TITLE
Micronaut-parent is still on 4.0.0-M1 on maven

### DISCUSSION THREAD

**Discussion by subash89** [micronaut-projects/micronaut-core#9574] [Q&A] [answered]
Hi Folks, 
I am trying to upgrade my app to micronaut 4.0.0 and saw micronaut-parent pom is still not released yet on 4.0.0. I assume now micronaut 4.0.0 release, this should also be ?

**Comment by graemerocher** [comment, marked as answer]
The group ID changed https://central.sonatype.com/artifact/io.micronaut.platform/micronaut-parent/4.0.0/versions

**Reply by subash89** [reply]
Got it. Thanks.

**Reply by sdelamo** [reply]
@subash89 we have instructions for upgrading https://micronaut.io/2023/05/09/upgrade-to-micronaut-framework-4-0-0/
